### DISCUSSION TITLE
ContentKeeperG3 HELP!

### DISCUSSION THREAD

**Discussion by zumwaltboi** [3kh0/ext-remover#692] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I need a bypass for the contentkeeper g3 extension... It blocks all bookmarklets, Most exploit links, and killcurly doesn't work for me. I also tried downgrading but when I booted from another version everything was still managed by my school admins. Please I need new (Contentkeeper bypassing) Exploits. PLEASE HELP!

**Comment by AshtonDavies** [comment, marked as answer]
Downgrading does not unenroll the Chromebook, but it does unlock patched exploits besides recovery ones like Sh1mmer. Reason is because enrollment and recovery is a separately updated partition that does not downgrade with the flash drive.

**Reply by Aka-but-nice** [reply]
I'm wondering why did he not downgrade to a version below 106, downgrading to v109 or lower would be nice as well since he'd be able to use the incognito exploit.

**Comment by obkduishinijf** [comment]
ltmeat, or downgrade to v102 or below and just use ltbeef
